script.js

```diff
@@ -1,4 +1,122 @@
+/*
+- player 1 rolls two dice and choose the order of the dice
+- player 2 rolls two dice and choose the order of the dice
+- compare player 1 and player 2 number, state the winner
+- reset the game
+*/
+
+var gameState = "player1Roll";
+var player1RollRecords = [];
+var player2RollRecords = [];
+var player1PointsRecord = [];
+var player2PointsRecord = [];
+
+// roll dice function
+var rollDice = function () {
+  let randomDiceNum = Math.floor(Math.random() * 6) + 1;
+  //console.log(randomDiceNum);
+  return randomDiceNum;
+};
+
+// player1 choose value function
+var chooseNumberFunction1 = function (input) {
+  if (input != 1 && input != 2) {
+    outputMessage = `Player 1, invalid option. 
+      For option 1, please key 1.
+      For option 2, please key 2.`;
+  }
+  if (input == 1) {
+    var player1Points =
+      String(player1RollRecords[0]) + String(player1RollRecords[1]);
+    player1PointsRecord.push(Number(player1Points));
+    outputMessage = `Player 1, your value is ${player1Points}`;
+    gameState = "player2Roll";
+  }
+  if (input == 2) {
+    var player1Points =
+      String(player1RollRecords[1]) + String(player1RollRecords[0]);
+    player1PointsRecord.push(Number(player1Points));
+    outputMessage = `Player 1, your value is ${player1Points}`;
+    gameState = "player2Roll";
+  }
```

Comment:
this is also fairly repetitive. since we passed the input validation already, we know that input is either 1 or 2, so we can just have a default value and check for the other, rather than checking for both cases explicitly. line 31-33 (and 38-40) can also just happen once outside of the if statement

```
var player1Points = String(player1RollRecords[0]) + String(player1RollRecords[1]);

if (input == 2) {
   var player1Points = String(player1RollRecords[1]) + String(player1RollRecords[0]);
}

player1PointsRecord.push(Number(player1Points));
outputMessage = `Player 1, your value is ${player1Points}`;
gameState = "player2Roll";
```